lib/client.js

```diff
@@ -518,8 +519,15 @@ Client.prototype.execute = function (query, params, options, callback) {
     params = typeof params !== 'function' ? params : null;
   }
   return utils.promiseWrapper.call(this, this.options, callback, function handler(cb) {
-    options = clientOptions.createQueryOptions(this, options);
-    this._innerExecute(query, params, options, cb);
+    let info;
```

Comment:
nit, rename `info` to `executionOptions`  or something like that.  (There are a few other instances of this)

Reply:
I thought I replaced them all :/
I'll fix these ones and double check others.